package.json

```diff
@@ -10,6 +10,7 @@
     "release": "scripts/release.sh"
   },
   "dependencies": {
-    "auto-changelog": "^2.4.0"
+    "auto-changelog": "^2.4.0",
+    "uuid": "^8.3.2"
```

Comment:
Where does this come from?

Reply:
by the installation of the uuid

Reply:
Yes, but why do we need it on top level?

Reply:
don't know, this was done automatically by yarn install.
what has to be changed on your opinion and how it have to be done?